ibm/service/kubernetes/resource_ibm_container_ingress_secret_opaque.go

```diff
@@ -73,6 +76,17 @@ func ResourceIBMContainerIngressSecretOpaque() *schema.Resource {
 				Computed:    true,
 				Description: "Status of the secret",
 			},
+			"update_secret": {
+				Type:        schema.TypeBool,
```

Comment:
we will have issue if you want to update_secret multiple times .
If user add this field for first time and mark update_secret = true it goes a head and update secret but next time when he wants to update again it will not show any diff to apply since update_secret = true  already exists in state file.

Generally for this kind of scenario we define "update_secret" as int when user want to change it he will just increment the value 

similar to https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/container_vpc_cluster#retry_patch_version

Reply:
fixed